Xamarin.Forms.Build.Tasks/XamlCTask.cs

```diff
@@ -55,7 +55,7 @@ public override bool Execute(out IList<Exception> thrownExceptions)
 				}
 
 				if (!string.IsNullOrEmpty(ReferencePath)) {
-					var paths = ReferencePath.Replace("//", "/").Split(';');
+					var paths = ReferencePath.Replace("//", "/").Split(';').Distinct();
```

Comment:
I think there is another spot higher for the `DependencyPaths` property  where calling `Distinct()` might help there too.

100ms is pretty good though, I think XamlC was 600ms overall I the simple project I saw the issue in.

Reply:
it's 100ms out of 4500 in my sample

Reply:
DependencyPaths is not used anymore, but I'll add the check

Reply:
done